cmake/RadiumSetupFunctions.cmake

```diff
@@ -484,7 +484,7 @@ ${CMAKE_INSTALL_PREFIX}/${RESOURCES_DESTINATION_DIR}/${RESOURCES_INSTALL_DIR}/${
     # Install in the install tree Identify the individual files( to preserve directory structure )
     # set the target properties
     get_target_property(resourceDir ${ARGS_TARGET} RADIUM_TARGET_RESOURCES_DIRECTORIES)
-    if(${resourceDir} STREQUAL "resourceDir-NOTFOUND")
+    if("${resourceDir}" STREQUAL "resourceDir-NOTFOUND")
```

Comment:
should it simply be 
```suggestion
    if(NOT "${resourceDir}")
```

Reply:
According to https://cmake.org/cmake/help/latest/command/if.html, yes, this should be simplified.

Reply:
I verified, this should not be simplified. I don't know why but doing this breaks the installation of the Resources (when more than one resource directory is associated with a target) of client libraries and plugins. I'll investigate more deeply later. Right now, we need to keep the proposal.